src/components/WebexDataProvider/README.md

```diff
@@ -1,3 +1,29 @@
-# Webex Activity Data Provider Component
+# Webex Data Provider
 
-LALLI HELP ME HERE PLEASE. MY BRAIN IS FRIED RIGHT NOW!
+Webex Data Provider creates a context that providers data adapters to its child components.
+
+## Usage
+
+1. Create an adapter for a data source.
+Webex Componets come with a JSON data adapter used to test components with static data.
+
+    ```js
+    import {WebexJSONAdapter} from '@webex/components';
+    import jsonData from './my-data';
```

Comment:
We should have more info on what the shape of the data object should look like.

Reply:
Added a link to the data directory.